lib/perl/Genome/Config/AnalysisProject/Command/AddConfigFile.pm

```diff
@@ -70,4 +79,28 @@ sub _mark_instrument_data_bridges {
     return 1;
 }
 
+sub _apply_tags {
+    my ($self, $profile_item) = @_;
+    return unless $self->tag;
```

Comment:
It would be good to have the test try using more than one tag.  This `return` isn't strictly necessary since the `for` loop will execute zero times if there are no tags.